### PR TITLE
Add CityMine

### PR DESCRIPTION
**Pull request by octolexgaming**
## Type of change

- [x ] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [ x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x ] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/manifest.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "name": "Citybuild", 
+      "color": "#ADD8E6",
+      "versions": "1.9<1.21"
+	},
+    "survival": {
+      "name": "Survival", 
+      "color": "#FFA500",
+      "versions": "1.9<1.21"
+	},
+    "grundstücke": {
```
Comment (DoJapHD) [inline minecraft_servers/manifest.json:19]: JSON Convention error; change to `grundstuecke`

**Review by DoJapHD — CHANGES_REQUESTED** [review]
Move all files into own folder called `citymine` inside of the `minecraft_servers` folder

**Comment by DoJapHD** [comment]
That's still not inside of a folder...

**Comment by octolexgaming** [comment]
I'm new to github how do I do that?

**Pull request closed without merging**